app/models/enrollment.js

```diff
@@ -103,5 +103,12 @@ module.exports = (sequelize, DataTypes) => {
         timestamps: true
     })
 
+    Enrollment.associate = (models) => {
+        Enrollment.belongsTo(models.Course)
+        Enrollment.belongsTo(models.Section)
+        Enrollment.belongsTo(models.User)
+        Enrollment.hasMany(models.Response)
+    }
```

Comment:
I think that's something that will be addressed if we get that far :) The idea is that it would make displaying grades faster to the instructor by doing calculations beforehand